core/server/utils/index.js

```diff
@@ -112,7 +112,8 @@ utils = {
     tokens: require('./tokens'),
     sequence: require('./sequence'),
     ghostVersion: require('./ghost-version'),
-    mobiledocConverter: require('./mobiledoc-converter')
+    mobiledocConverter: require('./mobiledoc-converter'),
+    markdownConverter: require('./markdown-converter')
```

Comment:
I wasn't sure if this is needed as it's not used via `utils.markdownConverter` anywhere but I followed the existing `mobiledocConverter` require (even though that's not used via `utils.mobiledocConverter`)